internal/scorer/config_test.go

```diff
@@ -198,17 +211,15 @@ func Test_buildCondition(t *testing.T) {
 
 		// Can't test the c.Not condition because algorithm.Condition is a func and can't be compared
 	}
-
-	for _, tt := range tests {
-		t.Run(tt.name, func(t *testing.T) {
-			got, err := buildCondition(tt.args.c)
-			if (err != nil) != tt.wantErr {
-				t.Errorf("buildCondition() error = %v, wantErr %v", err, tt.wantErr)
-				return
-			}
-			if !reflect.DeepEqual(got, tt.want) {
-				t.Errorf("buildCondition() got = %v, want %v", got, tt.want)
-			}
-		})
-	}
+	tt := tests[0]
+	t.Run(tt.name, func(t *testing.T) {
+		got, err := buildCondition(tt.args.c)
+		if (err != nil) != tt.wantErr {
+			t.Errorf("buildCondition() error = %v, wantErr %v", err, tt.wantErr)
+			return
+		}
+		if !reflect.DeepEqual(got, tt.want) {
+			t.Errorf("buildCondition() got = %v, want %v", got, tt.want)
+		}
+	})
```

Comment:
If the test is not in a loop the call to `t.Run` and the inner function are unneeded as well.

Also can your replace `tt` with `test` as above.

(I'm happy to leave the struct for the time being as well)

Reply:
I have removed the `t.Run` and changed `tt` to `test`.